card-app/src/getUsers.js

```diff
@@ -0,0 +1,19 @@
+import { UserCard } from "./UserCard";
+
+async function getUsers(userID) {
+    
+    const url = `https://reqres.in/api/users/${userID}`;
+
+    try {
+        const response = await fetch(url)
+        const data = await response.text();
+        const results = JSON.parse(data);
+        return results.data;
+    }
+    catch (err) {
```

Comment:
Also 👍 👍 to catching this - you could also`catch` in you component to display some type of error there if you wanted to as well.